### PR TITLE
fix(react): import SupportedStyles type from correct path

### PR DESCRIPTION
**Pull request by miluoshi**
## Current Behavior
Having incorrect import path for `SupportedStyles` type. There's no `packages/react/typings/index.d.ts` barrel file that would reexport style types.

## Expected Behavior
tsc doesn't throw error when typechecking schema

### CODE REVIEW

**Comment by uufukttas** [comment]
Hey @miluoshi! 
When will you merge this pr? 
I have errors for this problem on my project? Can you prioritize?

Thank you. :)

**Comment by miluoshi** [comment]
Hey @uufukttas I'm not a maintainer, I've just volunteered to fix the issue :) 

Someone from Nx team (@jaysoo ?) has to review it before it can be merged.

**Review by mandarini — APPROVED** [review]

**Pull request merged**
